npm-debug.log

```diff
@@ -0,0 +1,110 @@
+0 info it worked if it ends with ok
```

Comment:
Don't want to commit this file, let's get @cmontella's help to help you remove it from the PR (I have no idea how the GitHub app works, only git).